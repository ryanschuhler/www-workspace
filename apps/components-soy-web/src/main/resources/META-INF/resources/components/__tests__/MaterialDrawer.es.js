'use strict';

import MaterialDrawer from '../drawer/MaterialDrawer.es';

let component;

describe('MaterialDrawer', () => {
  afterEach(() => {
    if (component) {
      component.dispose();
    }
  });

  beforeAll(() => {
		window.Liferay = {};
  });

  it('should render', () => {
    component = new MaterialDrawer();

    expect(component).toMatchSnapshot();
  });
});
