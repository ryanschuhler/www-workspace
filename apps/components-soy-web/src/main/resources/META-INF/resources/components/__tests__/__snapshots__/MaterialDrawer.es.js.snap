// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MaterialDrawer should render 1`] = `
<nav class="mdc-permanent-drawer mdc-typography">
  <div class="mdc-permanent-drawer__content">
    <div class="mdc-persistent-drawer__toolbar-spacer"></div>
    <nav id="icon-with-text-demo" class="mdc-list">
      <a class="mdc-list-item mdc-permanent-drawer--selected" href="javascript:;">
        <i class="material-icons mdc-list-item__start-detail" aria-hidden="true">dashboard</i>Dashboard</a>
      <a class="mdc-list-item" href="javascript:;">
        <i class="material-icons mdc-list-item__start-detail" aria-hidden="true">settings</i>Settings</a>
    </nav>
  </div>
  <div class="mdc-permanent-drawer__toolbar-spacer"></div>
</nav>
`;
